TheFoodieNetwork/app/src/main/java/com/fbu/thefoodienetwork/CurrentUserUtilities.java

```diff
@@ -0,0 +1,196 @@
+package com.fbu.thefoodienetwork;
+
+import android.util.Log;
+
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseObject;
+import com.parse.ParseQuery;
+import com.parse.ParseRelation;
+import com.parse.ParseUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CurrentUserUtilities {
+    private final static String TAG = "CurrentUserUtilities";
+    public static List<String> currentUserFriendList;
+    public static List<String> currentUserReceivedFriendRequest;
+    public static List<String> currentUserSentFriendRequest;
```

Comment:
We should really make these private and have getters for them as this does not really adhere to the OOP pillar of "encapsulation".  The reason for this is that it will let another engineer mutate and set these variables when that might not really be what you want them to do.  Also, I don't think that we want these to be static.  If you're meaning for this class to only have one instance and for the information to be available throughout the app with the current user's information, then I would make this a singleton: https://www.tutorialspoint.com/design_pattern/singleton_pattern.htm

If you make this class a singleton, however, we need to make sure that we clean it up on logout so that we're not leaking user information across/between sessions.

We can talk more about this and how we should design a class like this, if you would like.